doc/enterprise/auth.md

```diff
@@ -0,0 +1,102 @@
+
+- [The behavior of pipelines when using access control](#behavior-of-pipelines-as-related-to-access-control)
+- [The behavior of a cluster when access control is de-activated or an enterprise token expires](#activation-code-expiration-and-de-activation)
+
+## Activating access control
+
+First, you will need to make sure that your cluster has Pachyderm Enterprise Edition activated (you can follow [this guide](deployment.md) to activate Enterprise Edition).  The status of the Enterprise features can be verified with `pachctl`:
+
+```
+$ pachctl enterprise get-state
+ACTIVE
+```
+
+Next, we need to activate the Enterprise access control features. This can be done with `pachctl auth activate ...`.  However, before executing that command, we should decide on at least one user that will have admin priviledges on the cluster. Pachyderm admins will be able to modify the scope of access for any non-admin users on the cluster.
+
+All users in Pachyderm are identified by their GitHub usernames.  Thus, if we wanted to activate access control on a cluster and set the GitHub user `dwhitena` as an admin, we would execute the following command:
+
+```
+$ pachctl auth activate --admins=dwhitena
+```
+
+Your Pachyderm cluster can have more than one admin if you like.  You would just need to specify them here as a comma separated list. 
+
+## Logging into Pachyderm
+
+Now that we have activated access control via `pachctl auth`, we can login to our cluster.  Let's login as the GitHub user `dwhitena` using `pachctl auth login <username>`.  When we execute this command, `pachctl` will provide us with a GitHub link to authenticate ourselves as the provided GitHub user, as shown below:
+
+```
+$ pachctl auth login dwhitena
+(1) Please paste this link into a browser:
+
+https://github.com/login/oauth/authorize?client_id=d3481e92b4f09ea74ff8&redirect_uri=https%3A%2F%2Fpachyderm.io%2Flogin-hook%2Fdisplay-token.html
+
+(You will be directed to GitHub and asked to authorize Pachyderm's login app on Github. If you accept, you will be given a token to paste here, which will give you an externally verified account in this Pachyderm cluster)
+
+(2) Please paste the token you receive from GitHub here:
+
+```
+
+When visiting this link in a browser, you will be presented with an option to "Authorize Pachyderm" (assuming that you haven't authorized Pachyderm via GitHub previously). Once you authorize Pachyderm, you will be presented with a Pachyderm user token:
+
+![alt tag](auth.png)
+
+Copy and paste this token back into the terminal, as requested by `pachctl`, and press enter.  You are now logged in to Pachyderm!  
+
+## Managing and updating user access
+
+Let's suppose that we create a repository call `myrepo` with the user `dwhitena`.  Because, the user `dwhitena` created this repository, `dwhitena` will have full read/write access to the repo.  This can be confirmed via the `pachctl auth check ...` command:
+
+```
+$ pachctl auth check owner myrepo
```

Comment:
@msteffen Is there a clear hierarchy with auth settings? Owner>writer>reader? If so, might makes to mention that here @dwhitena 

I ask because if I do `pachctl auth set JoeyZwicker writer myrepo`, I assume that means I'm also a reader? Or are those separate things?

Reply:
Ah, good point.  I will adjust.

Reply:
fixed below.

Reply:
Yup, owner > writer > reader (having a role implies having the roles below it)